CMakeLists.txt

```diff
@@ -22,4 +22,22 @@ configure_file("config.h.in" "config.h")
 add_definitions(-DHAVE_CONFIG_H -DNDEBUG)
 include_directories("${CMAKE_BINARY_DIR}")
 
+include_directories("src/common/public"
+	"src/DS/public"
+	"src/source/public"
+	"src/OVAL/public"
+	"src/XCCDF/public"
+	"src/XCCDF_POLICY/public"
+	"src/CVSS/public"
+	"src/CPE/public"
+	"src/CVE/public"
+	"${LIBXML2_INCLUDE_DIR}"
+	"${LIBXSLT_INCLUDE_DIR}"
+)
```

Comment:
This has side effects! We need to move all of this in utils/CMakeLists.txt and then do add_subdirectory("utils").